corehq/apps/geospatial/templates/map_visualization.html

```diff
@@ -37,33 +37,11 @@
 
 <div id="case-buttons" class="col-sm-12 col-md-12 col-lg-12">
     <div id="missing-gps-cases" style="display: inline;">
-        <button class="btn btn-danger" data-toggle="modal" data-target="#case-list"
+        <a class="btn btn-default" target="_blank" href="{% url 'gps_capture' domain %}"
                 data-bind="visible: casesWithoutGPS().length > 0">
             <span data-bind="text: casesWithoutGPS().length"></span>
             &nbsp;{% trans "Cases Missing GPS Data" %}
-        </button>
-        <div class="modal fade" id="case-list">
-            <div class="modal-dialog">
-                <div class="modal-content">
-                    <div class="modal-header">
-                        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span>
-                        <span class="sr-only">{% trans "Close" %}</span></button>
-                        <h4 class="modal-title">{% trans "Cases Missing GPS Data" %}</h4>
-                    </div>
-                    <div class="modal-body">
-                        <table class="table table-striped table-responsive">
-                            <thead><th>{% trans "Case Name" %}</th></thead>
-                            <tbody data-bind="foreach: casesWithoutGPS">
-                                <tr><td data-bind="html: link"></td></tr>
-                            </tbody>
-                        </table>
-                        <a class="btn btn-danger" target="_blank" href="{% url 'gps_capture' domain %}?data_type=case">
-                            {% trans "GPS Capture Page" %}
-                        </a>
-                    </div>
-                </div>
-            </div>
-        </div>
+        </a>
```

Comment:
Oh cool, I was worried if this lists gets too long.
It does look like we could simply the javascript code as well since this is not needed anymore.

Reply:
Looking at the JS code, I see `missingGPSModelInstance` is still being used in the missing cases button to show how many cases on a page are missing GPS data. I feel like we should keep this, as it could be useful to indicate to the user why some of the cases aren't on the map. @mkangia I'm curious to hear if you have different thoughts?

Reply:
There is a line that can be removed, which maps the case link to `casesWithoutGPS`, but this can be removed in [this](https://dimagi-dev.atlassian.net/browse/SC-2872) ticket.

Reply:
Ya, I think any improvement in javascript is out of the scope of this PR.
Good to remove anything that is redundant now since you removed the modal but don't seem reasonable to put any optimizations in this PR.